examples/stable_diffusion_v2/ldm/modules/train/optim.py

```diff
@@ -5,14 +5,14 @@
 from mindspore.nn.optim.adam import Adam, AdamWeightDecay
 
 
-def build_optimizer(model, optim, betas, weight_decay, lr):
+def build_optimizer(model, optim, lr, betas=[0.9, 0.999], weight_decay=1e-6):
```

Comment:
`betas=[0.9, 0.999]` - it is not good to use mutable default arguments. Besides, I already update `optim.py` in this [PR](https://github.com/mindspore-lab/mindone/pull/138/files#diff-492db109b632c3832b773d34b19699f56255889c4227e73d5a70f1ad9a69602b), just waiting it to be merged.